worker/src/core/loaders/message-worker/email.class.ts

```diff
@@ -51,27 +53,30 @@ class Email {
       })
   }
 
-  getMessages(
-    jobId: number,
-    rate: number
-  ): Promise<
-    {
-      id: number
-      recipient: string
-      params: { [key: string]: string }
-      body: string
-      subject: string
-      replyTo: string | null
-      from: string
-      campaignId: number
-    }[]
-  > {
-    return this.connection
-      .query('SELECT get_messages_to_send_email(:job_id, :rate);', {
+  async getMessages(jobId: number, rate: number): Promise<Message[]> {
+    interface MessageWithSenderEmail extends Message {
```

Comment:
Might want to move the interface definition somewhere else. Maybe `src/email/interfaces/index.ts`?

Reply:
Currently `senderEmail` is only used here (to check for the masthead) so it feels odd to separate this specific type from its usage

For now I've inlined the interface declaration (using an intersection type) so it's more clear that it's a temporary thing